library/firewall_lib.py

```diff
@@ -170,11 +170,12 @@
 
     FW_VERSION = firewall.config.VERSION
 
-    from firewall.client import FirewallClient, Rich_Rule, FirewallClientZoneSettings
 
+    from firewall.client import FirewallClient, Rich_Rule, FirewallClientZoneSettings
     HAS_FIREWALLD = True
 except ImportError:
     HAS_FIREWALLD = False
+    FirewallClient=None
```

Comment:
Hi Rich... I realized that I was making changes and the line above is necessary in order to properly mock out FirewallClient. Patching does not seem to work for some reason with conditional imports, if you know a better way to do this please let me know. Otherwise this is only I know to mock out FirewallClient.